### ISSUE TITLE
Redundant Ores Part II: Electric Boogaloo

### ISSUE THREAD

**Issue by Dayton-10** [Dayton-10/FreeCraft-III#7]
Thermal Expansion ores are still spawning alongside other same ores that are still enabled:
- Copper Ore (Thermal Expansion & Immersive Engineering)
Also likely:
- Silver Ore (Thermal Expansion & Immersive Engineering)
- Lead Ore (Thermal Expansion & Immersive Engineering)
- Nickel Ore (Thermal Expansion & Immersive Engineering)

Thermal Expansion doesn't seems to have a config, so we may have to disable Immersive Engineering ores instead :(

**Comment by Dayton-10** [comment]
Turns out Thermal Expansion world generation is now configured by a file inside the `<worldname>/serverconfig` folder, for each world save created (according to [this changelog](https://github.com/CoFH/Version/blob/master/thermal/changelog.md)).

I'm not yet sure how (if?) to include a change to this config file in the modpack itself, but it is at least something that _can_ be changed, if only server-side.

**Comment by Dayton-10** [comment]
Further research led to a similar issue being reported with Direwolf20's Building Gadgets mod [in this thread](https://github.com/Direwolf20-MC/BuildingGadgets/issues/462). The solution is given in [this comment from the above thread](https://github.com/Direwolf20-MC/BuildingGadgets/issues/462#issuecomment-609909871):
> I [sic] you want to make changes to a world you run then it's in `serverconfigs`. If you want to change the settings for the mod pack you're working on then you can copy the `buildinggadgets-server.toml` to the `/defaultconfig` folder and remove any values you're not changing and leave what you are changing. This in turn will tell forge to use those settings in preference to ours.
-- [MichaelHillcox](https://github.com/MichaelHillcox) (with added formatting)

I'll test this out myself and see what happens!

**Comment by Dayton-10** [comment]
After some testing, this does seem to work! I disabled copper, lead, silver, and nickel ore generation in the Thermal Expansion config file, copied the file to the `/defaultconfig` folder, and removed the parts of the config I didn't change. After using spectator mode, a cursory glance, and _legions of command block-controlled armor stands scouring the lands for rogue copper ore_ (it's a funny story) to check for the copper ore I disabled, I was able to fairly conclusively determine that the copper didn't generate. Just as it should be!

I'll implement these changes for the next modpack version and close this issue when it's complete.
